store/file/close_once.go

```diff
@@ -0,0 +1,79 @@
+package file
```

Comment:
Should we move it closer to interface definition as a helper implementation?

Reply:
That is something I wanted to discuss. closeOnce and validation are generic middleware around Accessor, so naturally they better fit into `eds` pkg, where Accessor is defined. However those seems to be useful only for file implementations and serves to solve files related problems. I also envision `eds` pkg as something that we want to extract into common repo/lib and those middleware are not necessarily needs to be placed into that repo/lib.